app/styles/_component-variables.scss

```diff
@@ -702,4 +702,7 @@ body {
   --dynamic-scan-modal-alert-background-color: var(--warn-light);
   --dynamic-scan-modal-alert-text-color: var(--warn-dark);
   --dynamic-scan-modal-alert-border-color: var(--warn-dark);
+
+  // variables for login-component
+  --ak-spinner-primary-contrast-text: var(--primary-contrast-text)
```

Comment:
variable name should start with component name eg: `login-component-button-loader-color`